docusaurus.config.js

```diff
@@ -47,6 +50,30 @@ const formatCopyright = () => {
   return `Copyright © Pants project contributors. ${repoLink} @ ${commitLink}.`;
 };
 
+const isPrerelease = (version) => {
+  const reference_dir = path.join(
+    "versioned_docs",
+    `version-${version}`,
+    "reference"
+  );
+  const helpAll = JSON.parse(
+    fs.readFileSync(path.join(reference_dir, "help-all.json"), "utf8")
+  );
+
+  const pantsVersion = helpAll["scope_to_help_info"][""]["advanced"].find(
+    (help) => help["config_key"] === "pants_version"
+  );
+
+  const hardcoded = pantsVersion["value_history"]["ranked_values"].find(
+    (value) => value["rank"] == "HARDCODED"
+  );
+
+  // Check if it's one of xx.xx.xx.dev0, xx.xx.xxa0, xx.xx.xxrc0
+  const rex = /^(\d+\.\d+\.\d+)(\.dev|a|rc)\d+$/;
```

Comment:
What about beta?

Reply:
I don't think I've ever seen a `b` suffix on any Pants release, but can add it.

Reply:
The fewer assumptions the better IMO. Thanks!